### PR TITLE
Add larger graph dataset, improve drawer, node search, layout adjustments

### PR DESCRIPTION
**Pull request by magiccookie**

### CODE REVIEW

File: src/graph/Graph.tsx [hunk 1]
```diff
@@ -83,20 +75,23 @@ const Graph2 = ({ data, config /* , chromaticScale */ }: Props) => {
...
 
-  const onSearchNode = () => {
-    if (!searchValue) {
+  const onSearchNode = ({ val, err, onError }: SearchArgs) => {
+    if (!val) {
       return
     }
 
-    const node = graph.nodes.find((node: Node) => node.data?.label.indexOf(searchValue) !== -1)
+    const node = graph.nodes.find((node: Node) => node.data?.label.indexOf(val) !== -1)
```
Comment (AndrewRatnikov) [inline src/graph/Graph.tsx:88]: What if we want to found a group of nodes that have the same subname. I suggest using a filter instead find

File: src/graph/Graph.tsx [hunk 2]
```diff
@@ -17,27 +18,21 @@ import {
   findDownstreamEdges,
 } from './graph-ops'
 
-const Graph2 = ({ data, config /* , chromaticScale */ }: Props) => {
+const GraphComponent = ({ data, config, layoutDirection = 'LR' /* , chromaticScale */ }: Props) => {
```
Comment (AndrewRatnikov) [inline src/graph/Graph.tsx:21]: I suggest putting 'layoutDirection' to 'config' to having the same model with other components

Comment (magiccookie) [reply src/graph/Graph.tsx:21]: Yes, we should put it to the config and a chromatic scale as well

Comment (magiccookie) [reply src/graph/Graph.tsx]: fixed

File: src/graph/Graph.tsx (hunk 1 above)
Comment (magiccookie) [reply src/graph/Graph.tsx]: applied some change, have a look

**Pull request merged**